File: magic8.py
#Magic 8 Ball simulator
import random

question = input("Ask a yes or no question:")

answer = random.randint(1,9)

if answer == 9:
    print('Magic 8 Ball: Yes - definitely.')
elif answer == 8:
    print('Magic 8 Ball: It is decidedly so.')
elif answer == 7:
    print('Magic 8 Ball: Without a doubt.')
elif answer == 6:
    print('Magic 8 Ball: Reply hazy, try again.')
elif answer == 5:
    print('Magic 8 Ball: Ask again later.')
elif answer == 4:
    print('Magic 8 Ball: Better not tell you now.')
elif answer == 3:
    print('Magic 8 Ball: My sources say no.')
elif answer == 2:
    print('Magic 8 Ball: Outlook not so good.')
else:
    print('Magic 8 Ball: Very doubtful.')

File: JS Unit/magic8.js
// More practice of if/else statements with a magic 8 ball
let answer = Math.floor(Math.random() * 9);

if (answer == 1) {
    console.log("Yes - definitely.");
} else if (answer == 2) {
    console.log("It is decidedly so.");
} else if (answer == 3) {
    console.log("Without a doubt.");
} else if (answer == 4) {
    console.log("Reply hazy, try again.");
} else if (answer == 5) {
    console.log("Ask again later.");
} else if (answer == 6) {
    console.log("Better not tell you now.");
} else if (answer == 7) {
    console.log("My sources say no.");
} else if (answer == 8) {
    console.log("Outlook not so good.");
} else {
    console.log("Very doubtful.");
}